index.bs

```diff
@@ -403,107 +410,100 @@ these steps:
 
 </div>
 
-## The {{FileSystemWriter}} interface ## {#api-filesystemwriter}
+## The {{FileSystemWritableFileStream}} interface ## {#api-filesystemwritablefilestream}
 
 <xmp class=idl>
 [Exposed=(Window,Worker), SecureContext]
-interface FileSystemWriter {
+interface FileSystemWritableFileStream {
```

Comment:
this should actually inherit from WritableStream, right?

Reply:
correct. Fixed.